src/main/java/io/sniffy/Sniffer.java

```diff
@@ -20,25 +26,25 @@
  */
 public final class Sniffer {
 
-    private Sniffer() {
+    private final static AtomicInteger executedStatementsGlobalCounter = new AtomicInteger();
 
-    }
+    private static final List<WeakReference<Spy>> registeredSpies = new LinkedList<WeakReference<Spy>>();
 
-    // Registered listeners (i.e. spies)
+    private Sniffer() {
 
-    private static final List<WeakReference<Spy>> registeredSpies = new LinkedList<WeakReference<Spy>>();
+    }
 
-    protected static synchronized WeakReference<Spy> registerSpy(Spy spy) {
+    static synchronized WeakReference<Spy> registerSpy(Spy spy) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/sniffy/sniffy/file/2495663886/issues/source?bid=3005960&fileBranchId=3034483#l37)